doc-content/enterprise-only/migration/migration-tool-niogit-7.4-proc.adoc

```diff
@@ -0,0 +1,217 @@
+[id='migration-tool-niogit-7.4-proc_{context}']
+
+= Using the migration tool on a `.niogit` directory
+
+The `.niogit` directory of your {CENTRAL} distribution contains all {CENTRAL} data. You can use the {CENTRAL} migration tool on your `.niogit` directory to prepare project data and system configurations for migration to {PRODUCT} 7.5.
```

Comment:
Probably mention something about not running BC 7.5 before doing this migration stuff?